recipes/oatpp/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "1.3.0-latest":
```

```suggestion
  "1.3.0_latest":
```
Modify version to make `Version("1.3.0_latest") > Version("1.3.0")`